R/infection_helpers.R

```diff
@@ -0,0 +1,136 @@
+#' Generate multiple infections with parameter distributions
```

Comment:
```suggestion
#' Generate multiple [daedalus::daedalus_infection] objects by sampling across distributions of input parameters
```

I **think** this is what is being done here. The use of `infection` without qualification makes it sound like an event within a simulation (to me at least)

Reply:
Yes that's right - I'll fix the docs.